src/app/zap-templates/zcl/data-model/chip/operational-state-rvc-cluster.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="OperationalStateEnum" type="ENUM8">
+    <cluster code="0x0061"/>
+    <item name="Stopped"        value="0x00"/>
+    <item name="Running"        value="0x01"/>
+    <item name="Paused"         value="0x02"/>
+    <item name="Error"          value="0x03"/>
+    <item name="SeekingCharger" value="0x40"/>
+    <item name="Charging"       value="0x41"/>
+    <item name="Docked"         value="0x42"/>
+  </enum>
+
+  <enum name="ErrorStateEnum" type="ENUM8">
+    <cluster code="0x0061"/>
+    <item name="NoError"                    value="0x00"/>
+    <item name="UnableToStartOrResume"      value="0x01"/>
+    <item name="UnableToCompleteOperation"  value="0x02"/>
+    <item name="CommandInvalidInState"      value="0x03"/>
+    <item name="FailedToFindChargingDock"   value="0x40"/>
+    <item name="Stuck"                      value="0x41"/>
+    <item name="DustBinMissing"             value="0x42"/>
+    <item name="DustBinFull"                value="0x43"/>
+    <item name="WaterTankEmpty"             value="0x44"/>
+    <item name="WaterTankMissing"           value="0x45"/>
+    <item name="WaterTankLidOpen"           value="0x46"/>
+    <item name="MopCleaningPadMissing"      value="0x47"/>
+  </enum>
+
+  <struct name="OperationalStateStruct">
+    <cluster code="0x0061"/>
+    <item name="OperationalStateID"    type="OperationalStateEnum" optional="false"/>
+    <item name="OperationalStateLabel" type="CHAR_STRING" optional="false" length="64"/>
+  </struct>
+
+  <struct name="ErrorStateStruct">
+    <cluster code="0x0061"/>
+    <item name="ErrorStateID" type="ErrorStateEnum" optional="false" />
+    <item name="ErrorStateLabel" type="CHAR_STRING" optional="false" length="64" isNullable="true"/>
+    <item name="ErrorStateDetails" type="CHAR_STRING" optional="true" length="64" isNullable="true"/>
```

Comment:
This one, however matches neither current spec nor https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7100 (which changed since you last looked at it, I bet.. which is the hazard of trying to target a work-on-progress).

Reply:
Same as above - that PR looks ready to be merged so I'll update these two fields accordingly.